### PR TITLE
[BULK] DocuTune - Add ms.custom tags for horizontals

### PR DESCRIPTION
**Pull request by alexbuckgit**
Add ms.custom tags for horizontals

CorrelationId: c81e3608-8fcd-4296-8dea-28909aaeded4
PointOfContact: @dbradish-microsoft

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [docs/architecture/containerized-lifecycle/design-develop-containerized-apps/build-aspnet-core-applications-linux-containers-aks-kubernetes.md](https://github.com/dotnet/docs/blob/2213022008f87e0fdcf530eda52112f4a8b8c4f2/docs/architecture/containerized-lifecycle/design-develop-containerized-apps/build-aspnet-core-applications-linux-containers-aks-kubernetes.md) | [Build ASP.NET Core applications deployed as Linux containers into an AKS/Kubernetes orchestrator](https://review.learn.microsoft.com/en-us/dotnet/architecture/containerized-lifecycle/design-develop-containerized-apps/build-aspnet-core-applications-linux-containers-aks-kubernetes?branch=pr-en-us-35516) |

### CODE REVIEW

**Comment by dbradish-microsoft** [comment]
#sign-off

This PR does not require ms.author review as it is part of a DocuTune bulk update to keep ms.custom values correct for horizontals (Azure CLI, Azure PowerShell, Terraform, Bicep, ARM Templates, Python, Java, JavaScript, SDK for Go etc.).

Please merge ASAP to avoid build conflicts.  Thank you.

**Review by IEvangelist — CHANGES_REQUESTED** [review]
This looks good, thank you. We'll :shipit: when you make the suggested changes.

**Comment by dbradish-microsoft** [comment]
@IEvangelist , I'm sorry, but i'm not following.  What changes are you requesting?

File: docs/azure/sdk/includes/enable-managed-identity-azure-cli.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-ms.custom: devx-track-azurecli
+ms.custom:
```
Comment (IEvangelist) [inline docs/azure/sdk/includes/enable-managed-identity-azure-cli.md:2]: This should either have a value, or be entirely removed. An empty value isn't valid, right?

**Comment by IEvangelist** [comment]
> @IEvangelist , I'm sorry, but i'm not following. What changes are you requesting?

Sorry, my original comment didn't come through - this is what I'm referring to https://github.com/dotnet/docs/pull/35516#pullrequestreview-1457216628

File: docs/azure/sdk/includes/enable-managed-identity-azure-cli.md (hunk 1 above)
Comment (dbradish-microsoft) [reply docs/azure/sdk/includes/enable-managed-identity-azure-cli.md:2]: @IEvangelist , ahhh, got it!  I'll close this PR and open a new one.  Thanks for the additional explanation.  Makes sense.

**Comment by dbradish-microsoft** [comment]
#please-close

**Comment by dbradish-microsoft** [comment]
#please-close

**Pull request closed without merging**